### PR TITLE
1.15 EOL Announcement

### PR DESCRIPTION
**Pull request by dhawton**
Please provide a description for what this PR is for.

1.15 EOL Annoucement

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by kfaseela** [comment]
Deploy page shows this as upgrade to 1.18. 
"At that point we will stop back-porting fixes for security issues and critical bugs to 1.15, so we encourage you to upgrade to the latest version of Istio (1.18). "

https://deploy-preview-12687--preliminary-istio.netlify.app/latest/news/support/announcing-1.15-eol/

**Comment by ericvn** [comment]
Thanks Daniel for getting this in early. We will wait to actually merge and cherry-pick on the 26th.

**Comment by dhawton** [comment]
> Thanks Daniel for getting this in early. We will wait to actually merge and cherry-pick on the 26th.

Sounds good

File: content/en/news/support/announcing-1.15-eol/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Support for Istio 1.15 ends on March 28th, 2023
+subtitle: Support Announcement
+description: Upcoming Istio 1.15 end of life announcement.
+publishdate: 2023-02-26
+---
+
+According to Istio's [support policy](/docs/releases/supported-releases#supported-releases/), minor releases like 1.15 are supported until six weeks after the N+2 minor release (1.17 in this case). [Istio 1.17 was released on February 13th](/news/releases/1.17.x/announcing-1.17/), and support for 1.15 will end on March 28th, 2023.
+
```
Comment (johnma14) [inline content/en/news/support/announcing-1.15-eol/index.md:9]: Istio 1.17 was released on Feb 14th

**Review by jacob-delgado — APPROVED** [review]

**Review by ericvn — APPROVED** [review]
We can merge this here today, and then we will merge the cherry-pick on the date mentioned in the announcement, 2/26.

**Pull request merged**